libkbfs/crypto_common.go

```diff
@@ -463,7 +463,11 @@ func (c CryptoCommon) DecryptBlock(encryptedBlock EncryptedBlock, key BlockCrypt
 		return err
 	}
 
-	return c.codec.Decode(encodedBlock, &block)
+	err = c.codec.Decode(encodedBlock, &block)
```

Comment:
Don't rely on it anymore, but it still seems like it gives strictly more information when debugging issues here.  If you think it's confusing I can take it out though.